internal/pkg/template/templates/app/app.yml

```diff
@@ -30,7 +30,7 @@ Conditions:
 Resources:
   AdministrationRole:
     Metadata:
-      'aws:copilot:description': 'A StackSet admin role assumed by CloudFormation to manage regional stacks'
+      'aws:copilot:description': 'A StackSet admin role {{- if .PermissionsBoundary}} with permissions boundary {{.PermissionsBoundary}} {{- end}} assumed by CloudFormation to manage regional stacks'
```

Comment:
This seems to repeat a lot...and also for upcoming new roles we need this as well. Maybe consider making it a template function?

Reply:
oh nvm...